app/contracts/project_queries/publish_contract.rb

```diff
@@ -27,7 +27,13 @@
 #++
 
 module ProjectQueries
-  class PublishContract < BaseContract
+  class PublishContract < ::ModelContract
```

Comment:
Did you omit the `frozen_string_literal: true` magic comment on purpose? If not, this would be a good opportunity to add it.